src/mtcp/Makefile.in

```diff
@@ -88,8 +88,12 @@ default: build
 libs: build
 build: $(targetdir)/bin/$(MTCP_RESTART)
 
+# TODO(kapil): Check if -ttext-segment value is appropriate for x86.
+# The other sections (bss, rodata, etc.) will follow immediately after text.
+DEBUG_FLAGS= -Wl,-Ttext-segment=11100000 #-Wl,-Trodata-segment=445000000
```

Comment:
Shouldn't the address be written in hex?  Something like:  `-Wl,-Ttext-segment=0x7000000`
When debugging it's easier if this is a round number in hex.

Reply:
The address is hex, but without a `0x` prefix. The compiler address-related flags expect hex addresses without a `0x` prefix.